catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -484,6 +483,57 @@ protected DAGQueryResult getData(long queryTime) {
                                 .is()
                                 .after()
                                 .date(new Date(queryTime)));
+
+                //Always need to ask for the DEFAULT_TAG or we get non-resource metacards
+                Filter resourceFilter = filterBuilder.allOf(parsedFilter, filterBuilder.attribute(
+                        Metacard.TAGS)
+                        .is()
+                        .like()
+                        .text(Metacard.DEFAULT_TAG));
+
+                //Default for NSILI is to include OBSOLETE (deleted) items
+                if (!query.bqs_query.toLowerCase()
```

Comment:
Should probably put this logic in a separate method since it is repeated in the else clause. I noticed multiple other classes use this too, maybe could pull this out into a class with a static method.